summit-pitch.md

```diff
@@ -0,0 +1,37 @@
+# Summit pitch
+
+Questions: who shall be the person that pitches?
+
+## Title
+*The title of your talk/discussion topic. This will appear in the Language Summit schedule on the conference website.*
+
+Fearless Concurrency in Python
+
+
+## Key discussion item
+
+> *What is the "thing" you want to discuss in front of mostly Python core developers? What decision to be made? What are your questions, and proposed solutions? This should fit in 10 minutes, to allow for 20 minutes of discussion. Therefore this should be very focused. This field will only be read by Language Summit co-chairs to determine the talk acceptance. You can include a rough outline, useful links, sample discussions, etc.*
+
+
+We want to discuss a dynamically checked concurrency/ownership model for Python.
+The concurrency model is based on "regions" and "deep immutability", and allows existing Python data structures to be used in multi-threaded programs.
+
+The presentation will be based on the ideas presented in the PLDI'25 paper (link to be added)
+and prototyping work done on a fork of the CPyton interpreter.
+
+The aim of the presentation is to get feedback on the ideas, and to discuss the implications of the model for Python.
+Getting something like this into Python would be a big change, and we want to discuss this with the core developers.
```

Comment:
Do we want to say something about how we expect to drive most of the implementation work?

Reply:
I think mentioning that you already have a rough plan of 4-5 PEPs (each of which incrementally improves Python) would probably be worth mentioning.  That shows the start of the concrete plan.

Reply:
Ok, great! We were saying earlier today that we wanted to show that in the presentation. I’ll add it here. Thanks!

Reply:
@mdboom added now!